database/seeders/UsersTableSeeder.php

```diff
@@ -49,7 +49,7 @@ public function run(): void
                 'username'          => config('unit3d.owner-username'),
                 'email'             => config('unit3d.default-owner-email'),
                 'email_verified_at' => now(),
-                'group_id'          => 10,
+                'group_id'          => 9,
```

Comment:
Can we keep this as is? I just found a hardcoded check for `group_id = 10` the other day for checking if a user is owner and hope there's no more, but better be safe than sorry.

Reply:
This was duet to mistake in PR #3502 . A commit has been pushed to 7.x.x to revert that. This can be put back to 10 .